src/frequenz/sdk/timeseries/_moving_window.py

```diff
@@ -138,6 +138,16 @@ def __init__(  # pylint: disable=too-many-arguments
         self._resampler_sender: Sender[Sample] | None = None
         self._resampler_task: asyncio.Task[None] | None = None
 
+        """The number of samples that have been received."""
+        self.count_samples = 0
+        """The number of samples to wait for before
+        the wait_for_samples method triggers."""
+        self.wait_for_num_samples = 0
```

Comment:
docstrings need to be below the item they are docing.
```suggestion
        self.count_samples = 0
        """The number of samples that have been received."""
        self.wait_for_num_samples = 0
        """The number of samples to wait for before
        the wait_for_samples method triggers."""
```

Reply:
done